driver/razerkbd_driver.h

```diff
@@ -36,6 +36,7 @@
 #define USB_DEVICE_ID_RAZER_BLADE_2018_BASE 0x023b
 #define USB_DEVICE_ID_RAZER_BLADE_2018_MERCURY 0x0240
 #define USB_DEVICE_ID_RAZER_BLADE_MID_2019_MERCURY 0x0245
+#define USB_DEVICE_ID_RAZER_BLADE_PRO_2019 0x0234
```

Comment:
Please move your device up after `USB_DEVICE_ID_RAZER_BLADE_2018` as well to have them sorted